tests/acceptance/01_vars/01_basic/os_name_human.cf

```diff
@@ -0,0 +1,54 @@
+body common control
+{
+  bundlesequence => { "test", "check" };
+}
+
+bundle agent test
+{
+  meta:
+      "description" -> { "CFE-3569" }
+        string => "Test for expected values of variable sys.os_name_human";
+
+  # Order is important, the last variable assignment wins.
+  # E.g. prefer Ubuntu over Debian
+  vars:
+    any::
+      "actual" string => "$(sys.os_name_human)";
+    debian::
+      "expected" string => "Debian";
+    ubuntu::
+      "expected" string => "Ubuntu";
+    redhat::
+      "expected" string => "RHEL";
+    centos::
+      "expected" string => "CentOS";
+    fedora::
+      "expected" string => "Fedora";
+    aix::
+      "expected" string => "AIX";
+    hpux::
+      "expected" string => "HPUX";
+    suse::
+      "expected" string => "SUSE";
+    macos::
+      "expected" string => "macOS";
+    windows::
+      "expected" string => "Windows";
+    freebsd::
+      "expected" string => "FreeBSD";
```

Comment:
Since macOS is based on FreeBSD I would put it after, but I'm pretty sure `freebsd` is not defined on mac so it shouldn't make a difference.